src/Navbar/style.scss

```diff
@@ -0,0 +1,91 @@
+@import './_variables.scss';
+
+header {
+  display: flex;
+  width: 100%;
+  margin-left: 0;
+}
+
+.header_logo {
+  margin-top: 4px;
+  margin-left: 5%;
+}
+
+nav {
+  margin-top: 46px;
+  margin-left: 12%;
+}
+
+.navigation {
+  a {
+    color: $white;
+    text-align: center;
+    padding: 22px 24px;
```

Comment:
I believe u should have a lot of warnings in the lint. Do u?

Reply:
now all clear